bitbots_vision/src/bitbots_vision/vision_modules/color.py

```diff
@@ -194,37 +205,51 @@ class PixelListColorDetector(ColorDetector):
         'field_color_detector_path'
     """
 
-    def __init__(self, debug_printer, package_path, config):
-        # type:(DebugPrinter, str, dict, bool) -> None
+    def __init__(self, config, package_path, pub_field_mask_image=None):
+        # type:(dict, str, rospy.Publisher) -> None
         """
         Initialization of PixelListColorDetector.
 
-        :param DebugPrinter debug_printer: debug-printer
+        :param dict config: dictionary of the vision node configuration parameters
         :param str package_path: path of package
-        :param dict config: vision config
+        :param rospy.Publisher pub_field_mask_image: Publisher of field mask images (Default: None)
         :return: None
         """
-        super(PixelListColorDetector, self).__init__(debug_printer)
-        self.bridge = CvBridge()
+        self.package_path = package_path
```

Comment:
If I get it right, this requires a hardcoded 'bitbots-vision' as a package to get the package path. It would cause conflicts if `color.py` is used outside of the scope of the bitbots-vision.